app/assets/stylesheets/sephora_style_guide/mixins/_border_radius.scss

```diff
@@ -0,0 +1,33 @@
+@mixin border-radius($radius: $border-radius) {
```

Comment:
AFIK we rarely have designs with rounded borders so I didn't include these initially.